source/elements/oneMKL/source/domains/rng/host_api/rng-host-routines.rst

```diff
@@ -0,0 +1,48 @@
+.. SPDX-FileCopyrightText: 2019-2020 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_rng_manual_offload_routines:
+
+Random Number Generators Host Routines
+======================================
+
+.. rubric:: Structure
+
+RNG domain contains two classes types:
```

Comment:
it makes sense to combine host and device routines and move it to rng-overview

Reply:
The thing is that we need this file to nest all the topics. I can move common text part to separate file and just `.. include::` it

Reply:
agreed to postpone